nima_core/storage/migrations/manager.py

```
@@ -0,0 +1,174 @@
+
+            if dry_run:
+                print("  [DRY RUN]")
+                continue
+
+            try:
+                success, elapsed = self._executor.execute_migration(migration)
+                if success:
+                    print(f"  Applied in {elapsed}ms")
+                    applied.append(migration)
+                else:
+                    print(f"  Failed")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)